src/browser_authentication_app_skeleton/src/actions/mixins/auth/test_backdoor.cr

```diff
@@ -1,13 +1,21 @@
+# When testing you can skip normal sign in by using `visit` with the `as` param
+#
+#     user = UserBox.create
+#     flow.visit Me::Show, as: user
+#
+# The module only works in the 'test' environment.
 module Auth::TestBackdoor
-  macro included
-    before test_backdoor
-  end
+  {% if env("LUCKY_ENV") == "test" %}
```

Comment:
Does not work because LUCKY_ENV is set at runtime in spec_helper, so is not available at this point :(